matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/send/SendService.kt

```diff
@@ -56,6 +56,16 @@ interface SendService {
      */
     fun sendFormattedTextMessage(text: String, formattedText: String, msgType: String = MessageType.MSGTYPE_TEXT): Cancelable
 
+    /**
+     * Method to quote an events content.
+     * @param quotedEvent The event to which we will quote it's content.
+     * @param text the text message to send
+     * @param msgType the message type: MessageType.MSGTYPE_TEXT (default) or MessageType.MSGTYPE_EMOTE
+     * @param autoMarkdown If true, the SDK will generate a formatted HTML message from the body text if markdown syntax is present
+     * @return a [Cancelable]
+     */
+    fun sendQuotedTextMessage(quotedEvent: TimelineEvent, text: String, autoMarkdown: Boolean = false): Cancelable
```

Comment:
There is a problem with our current "Quote" action on Element Android. User should be able to edit the quoted text, as per EleWeb.
I should create an issue about it, if it does not exist yet.

So I'm not sure that this new API is necessary.

Reply:
Editable quotes won't really work with the markdown preference disabled. As it's kind of like any other message at that point and we won't parse the block quote. Maybe we could further explore having a simple/advanced mode and explore further can we can more fine grained control over commonmark or another parser.